bundles/org.openhab.binding.daikin/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -147,6 +173,172 @@
 			</options>
 		</state>
 	</channel-type>
+	
+	<channel-type id="acunit-cmpfrequency" advanced="true">
+		<item-type>Number</item-type>
+		<label>Compressor Frequency</label>
+		<description>Current compressor frequency</description>
+		<state readOnly="true" pattern="%.0f%%"></state>
```

Comment:
You are displaying the frequency as a percentage?

Reply:
Daikin calls it frequency, but it is from my understanding a percentage.

Reply:
I leave it with Dimensionless and show percentage, good point.